app/javascript/components/field-array/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row, Col, Button, FormGroup } from 'patternfly-react';
+
+import FieldArrayItem from './field-array-item';
+
+const FieldArray = ({
+  FieldProvider,
+  FieldArrayProvider,
+  arrayValidator,
+  label,
+  fields,
+  itemDefault,
+  formOptions,
+  addText,
+  ...rest
+}) => {
+  const { name } = rest.input;
+  const colsize = Math.floor(10 / fields.length);
+
+  const header = (
+    <Row>
+      <div className="field-array-header">
+        { fields.map(({ label }, index) => (
+          <Col xs={colsize} key={index}><label>{ label }</label></Col>
+        )) }
+        <Col xs={2}><label>{ __('Actions') }</label></Col>
+      </div>
+    </Row>
+  );
+
+  return (
+    <FieldArrayProvider name={name} validate={arrayValidator}>
+      { cosi => (
```

Comment:
:D :czech_republic: or :it: ?

Reply:
I don't know, copied from https://data-driven-forms.org/ :laughing: so I guess it's :czech_republic: